src/tasks.py

```diff
@@ -45,6 +47,28 @@ def ami_for_project(pname):
     # good for figuring out filters 
     #print results[0].__dict__
 
+@task
```

Comment:
oh - you just moved the ami stuff here. um. was there a good reason for the move? it's not part of this builder, but there was some code for using `packer` to build vagrant boxes that may come back around